### ISSUE TITLE
Solo Strike crashes to desktop

### ISSUE THREAD

**Issue by nk1tra** [DrNoLife/Destiny-2-Solo-Enabler#23]
I have an issue where whenever I play the strike that I'm soloing it crashes to desktop. I do not know how to fix this. It only occurs when I'm soloing the strike, not regular or in any other game mode in Destiny 2. 

Update: Tested it out more. The crash only seems to occur in Vanguard playlisted strikes. Individually selected strikes experience no crashes.

**Comment by hersek138** [comment]
yea Ive been having the same problem recently.

**Comment by psdemeter** [comment]
It locks up then crashes to desk top everywhere

**Comment by Piloh-Jolie-Schitt** [comment]
Both I and a friend of mine are having the same issue. For me at least, the crashes weren't restricted to strikes and could happen at any time. The program still does what it's meant to do - I'm able to do open-world, strikes, etc. on my own. I don't have an exact number yet but he and I both experience the crashes about 20 to 30 minutes after enabling the program.

**Comment by cyphacipher** [comment]
Same issue, happened on 3 strikes today. Normally I play to the end of strike, then turn off the solo enabler and let others join for boss. But if it takes me a while to get to the boss, the game will crash. Screen freezes, audio loops the last 3-4 seconds until I force close the game. Only happens with playlist strikes, with enabler on. No enabler, and I can play everything normally, no crash. I opened task manager on the last strike, and did not notice any significant memory usage or anything. Could it be that the game is checking for the enabler now?

**Comment by RELynn21** [comment]
bump.
experiencing same issue, destiny freeze and crashes while doing a free roam, new Dares of Eternity event, and vanguard playlist. freeze and crashes occurred after 10-15 mins the program enabled.

**Comment by Percdye7** [comment]
Can confirm, Game Crashes after some time.

Weirdly, the game is still running as a Process after crashing. so you'll have to end the task

**Comment by Hedronmx** [comment]
I suspect it has to do with battleye

**Comment by Scouter953** [comment]
This is still an issue to this day. Super frustrating to get through 3/4 of an activity just to get kicked.

**Comment by MrRudiculous** [comment]
I too am having this issue, game closes halfway through the strike with no errors but destiny2.exe is still running in task manager, using 0% cpu/gpu but still the same amount of ram ~3gb. However according to windows, this doesn't actually register as a game crash so no crash logs in event viewer for destiny2.exe. I've checked around the other events and noticed that whenever destiny does close, 'BEDaisy' occurs after i end end the process in task manager.

Destiny 2 crash logs that are found in the localappdata/temp folder after a 'crash' always gives me an error that says networking:transport:steam: oom unable to allocate xxxx bytes, that number varies from 10 - 20000. I don't know what any of that means but it is the cause for every single crash_info.

**Comment by Dante2202** [comment]
I can confirm this crashes steam constantly and is not stable any longer. I have tried numerous times and running this always crashes destiny on steam. It also doesn’t crash it in a normal way, you have to force close steam and reload it to restart the game. Please look into this, it is unusable in its current form.

**Comment by DrNoLife** [comment]
I can't reproduce this behavior at all. For me, it works as it always has, works if I try to go solo onto a planet, strike or a Vanguard Ops.

**Comment by MrRudiculous** [comment]
The program/script works, I can load into any activity solo, it's just that after a certain amount of time (sometimes 5 mins, sometimes 20 mins) the game window just closes but not the process and steam still says it's running. 

Up until yesterday this only happened in vanguard playlist, specifically strikes as bungie have included previous seasons battlegrounds as strikes and those work fine e.g battlegrounds - oracle. For some reason I crashed in the exact same way as I described above whilst customizing my appearance in orbit, it was maybe around 2 hours after I switched on the program/script.

**Comment by DrNoLife** [comment]
Well, I'll give it a try again today, and see if the crash will happen.
Just to be clear, it's only crashing after some time, like after 30+ minutes? If that's the case, then I wonder if maybe allowing traffic to flow through for some seconds, each 15 minutes might fix it (i.e. turning it off and on on a schedule)? If that would be a "fix", then it should be easy enough to implement. 

But as mentioned before, I'll give it another try later and see what happens.

**Comment by Dante2202** [comment]
This happens about 15-20 minutes for me when trying to beat the moon campaign “a mysterious disturbance”. I set solo lobby on before I start, after I get through first initial encounter after about 15 minutes or so in the second encounter the game always crashes there. In steam it still shows it’s running but the video freezes and the window has completely vanished it is nowhere to be seen. But steam still shows the game running somehow.

**Comment by MrRudiculous** [comment]
Yes only after some time within the vanguard strike itself launched from the playlist. There are some rare cases where it doesn't happen but that is literally 1 in 10 but eventually it does crash

**Comment by MrRudiculous** [comment]
> This happens about 15-20 minutes for me when trying to beat the moon campaign “a mysterious disturbance”. I set solo lobby on before I start, after I get through first initial encounter after about 15 minutes or so in the second encounter the game always crashes there. In steam it still shows it’s running but the video freezes and the window has completely vanished it is nowhere to be seen. But steam still shows the game running somehow.

Hmm that mission is solo without the program running apart from the 1st part with the plate capturing, it doesn't matchmake per se but the game puts you into another an instance of that mission that other people are also running, once you get to the tank area, everyone disappears and you are alone for the rest of the mission. 

The errors could be related to the game trying to find you some help or others doing the same for parts that can contain other players but that's just a guess since it only happens in strikes for me, works perfectly in the new wellspring activity and psiops playlist

**Comment by DrNoLife** [comment]
Well update. Guess I just got lucky the other times I tested it, because when I did it again today, I experienced the crashing issue.
Unfortunately I don't really know how to fix this issue at the moment. I tried to be less strict in regards to the ports being blocked, but that amounted to nothing. Neither did my other idea, where the program would turn itself off for a few seconds every ten minutes. 

So it seems like Bungie did in fact change something in regards to the Vanguard Ops playlist. I didn't check the other playlists, as each run took 30 minutes before a crash. So it takes a long time to check just a single change. 

One other user mentioned he has a powershell script he claims to be working, here's a link to that: https://github.com/Percdye7/destiny2solo-PS
Whether or not it works, I don't know, I tried changing the ports being blocked to the same ones he's using, but alas changed nothing. But well, I guess it's worth a try?

Only thing I can do is try to run Wireshark to see if they changed it up so there's a specific port that's being used to send _something_ essential for either the game or BattlEye. Other than that, there's not much that can be done.

**Comment by MrRudiculous** [comment]
Thanks for the update and the effort to try and resolve this, it is a shame that bungie has changed something (I still think it's battleye since this only started to happen after it was implemented) , I will try the script you mentioned but before my 1st comment I too was just using a powershell script to enable this, thought I'd try your program to see if it made any difference

**Comment by Dante2202** [comment]
Thanks for the update as well.  I can also confirm I tried the method on Xbox to get into solo mode which also still works.  What is interesting is that the game also crashes on Xbox at the same point for me.  I would think it's gotta be something on Bungie's end causing this.

**Comment by HeliumSynth** [comment]
Ty for the update. I have no idea if this script seems to be the issue as well but I used it and its not only strikes that made me crash but Dares of Eternity as well.

**Comment by Geist4587** [comment]
Please please watch my videos I don't want any followers I need the help of the masses Bungie the Destiny 2 developers have been hacking my HP Omen PC for over a year I've asked for help everywhere Facebook Instagram YouTube and Twitter put me in incognito so the truth  It doesn't come out, the sad thing is I haven't even paid it off completely and they don't stop hacking me, please distribute my videos, help me, I've been fighting Bungie for a long time with depression.  https://youtube.com/shorts/feRDkmBOyA4?si=GoWKFnyxqIi-eWkg
